static/app/components/events/interfaces/spans/waterfallModel.tsx

```diff
@@ -41,10 +43,11 @@ class WaterfallModel {
     event: Readonly<EventTransaction | AggregateEventTransaction>,
     affectedSpanIds?: string[],
     focusedSpanIds?: string[],
-    hiddenSpanSubTrees?: Set<string>
+    hiddenSpanSubTrees?: Set<string>,
+    traceInfo?: TraceInfo
   ) {
     this.event = event;
-
+    this.traceInfo = traceInfo;
```

Comment:
Does this need to be a class property if we're only using it in the one method?

Reply:
It doesn't, I passed it as an optional prop to` getWaterfall` and `generateBounds` instead.